docs/apache-airflow-providers-apache-kafka/index.rst

```diff
@@ -54,48 +54,8 @@ Content
     PyPI Repository <https://pypi.org/project/apache-airflow-providers-apache-kafka/>
     Installing from sources <installing-providers-from-sources>
 
-.. toctree::
-    :maxdepth: 1
-    :caption: Commits
-
-    Detailed list of commits <commits>
-
-
-Package apache-airflow-providers-apache-kafka
-------------------------------------------------------
-
-`Apache Kafka <https://kafka.apache.org/>`__
-
-
-Release: 1.0.0
-
-Provider package
-----------------
-
-This is a provider package for ``apache.kafka`` provider. All classes for this provider package
-are in ``airflow.providers.apache.kafka`` python package.
-
-Installation
-------------
-
-You can install this package on top of an existing Airflow 2 installation (see ``Requirements`` below)
-for the minimum Airflow version supported via ``pip install apache-airflow-providers-apache-kafka``.
-
-Requirements
-------------
-
-===================  ==================
-PIP package          Version required
-===================  ==================
-``apache-airflow``   ``>=2.3.0``
-``confluent-kafka``
-===================  ==================
-
-.. include:: ../../airflow/providers/apache/kafka/CHANGELOG.rst
-
```

Comment:
Why this is removed?

Reply:
It's been duplicated due to conflict resolution likely (see below)

Reply:
This part is automatically (re)generated every time you prepare docs, but it was also put accidentally above the comment that marks the end of manually maintained part of the index.

Reply:
no no - it's good you removed it :). It should not be there.